### PR TITLE
No formatting necessary.

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

Ref: https://github.com/cosmos/ibc-go/pull/4934#discussion_r1375906537

closes: #XXXX

### Commit Message / Changelog Entry

```text
chore: do not call Wrapf when not required.
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: modules/light-clients/08-wasm/types/proposal_handle.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func (cs ClientState) CheckSubstituteAndUpdateState(
 
 	_, ok := substituteClient.(*ClientState)
 	if !ok {
-		return errorsmod.Wrapf(
```
Comment (damiannolan) [inline modules/light-clients/08-wasm/types/proposal_handle.go:31]: we could remove the `fmt.Sprintf()` below too :D 

And then actually use `Wrapf()` here correctly!

**Review by damiannolan — APPROVED** [review]
Nice one!

File: modules/light-clients/08-wasm/types/proposal_handle.go (hunk 1 above)
Comment (DimitrisJim) [reply modules/light-clients/08-wasm/types/proposal_handle.go:31]: glorious

Comment (damiannolan) [reply modules/light-clients/08-wasm/types/proposal_handle.go:31]: love the commit msg 😆

**Review by crodriguezvega — APPROVED** [review]

**Pull request merged**